src/collections/Form/index.d.ts

```diff
@@ -173,7 +173,7 @@ interface FormGroupProps {
 
 export const FormGroup: React.ComponentClass<FormGroupProps>;
 
-interface FormInputProps extends InputProps, ReactFormEvents<HTMLInputElement>, ReactFocusEvents<HTMLInputElement> {
+interface FormInputProps extends InputBaseProps, ReactFormEvents<HTMLInputElement>, ReactFocusEvents<HTMLInputElement> {
```

Comment:
I think we can just remove `ReactFormEvents<HTMLInputElement>` and `ReactFocusEvents<HTMLInputElement>` there, because we don't [expose](https://github.com/Semantic-Org/Semantic-UI-React/issues/1072#issuecomment-268848820) non SUIR props (#1072).